session/session.go

```diff
@@ -2678,7 +2678,13 @@ func (s *session) Auth(user *auth.UserIdentity, authentication, salt []byte) err
 		}
 		return err
 	}
-	s.sessionVars.ResourceGroupName = info.ResourceGroupName
+
+	// If tidb_resource_control_enable is disabled, set resource group to default
+	if variable.EnableResourceControl.Load() {
+		s.sessionVars.ResourceGroupName = info.ResourceGroupName
```

Comment:
```suggestion
		s.sessionVars.ResourceGroupName = strings.ToLower(info.ResourceGroupName)
```
Should convert to lower string so other following usage won't need to care about the capitalization.